seeds/test/UserSeed.js

```diff
@@ -0,0 +1,8 @@
+module.exports = [
```

Comment:
did you try `export { seeds }`? maybe you need a named parameter... where are you using UserSeeds? I did not found the line where you import the seeds...